docs/development/pigs/pig-025.rst

```diff
@@ -0,0 +1,96 @@
+.. include:: ../../references.txt
+
+.. _pig-xxx:
+
+
+**********************************
```

Comment:
the lenght of the '***' should exactly match the text lenght, if I do not mistaken (rst formatting).
The same applies below for the '===='

Reply:
I tried to change the length of === and *** but found no noticable difference in the appearance of the file.